lib/import_pipelines.js

```diff
@@ -12,20 +12,75 @@ var through = require( 'through2' );
 var logger = require( 'pelias-logger' ).get( 'openaddresses' );
 var peliasModel = require( 'pelias-model' );
 var peliasDbclient = require( 'pelias-dbclient' );
-var cleanup = require( './cleanup' );
+var cleanup = require( './cleanupStream' );
 
+/*
+ * Return true if a record has all of LON, LAT, NUMBER and STREET defined
+ */
 function isValidCsvRecord( record ){
-  return [ 'LON', 'LAT', 'NUMBER', 'STREET' ].reduce( function ( acc, prop ){
-    return acc && record[ prop ].length > 0;
-  }, true);
+  return [ 'LON', 'LAT', 'NUMBER', 'STREET' ].every(function(prop) {
+    return record[ prop ].length > 0;
+  });
 }
 
-/**
- * Used to track the UID of individual records passing through the stream
- * created by `createRecordStream()`. Not bothering with a closure for
- * readability. See `peliasModel.Document.setId()` for information about UIDs.
+/*
+ * Create a through2 stream to filter out invalid records
+ */
+function createValidRecordFilterStream() {
```

Comment:
I think it generally reads cleaner if each stream is in its own module, even if really simple.